src/www/candidatelist.js

```diff
@@ -0,0 +1,138 @@
+(function() {
+    const {invitegroup} = window.commjs;
+
+    window.addEventListener("DOMContentLoaded", () => {
+
+    });
+
+    /**
+     * @summary updates all dynamic html elements of the page
+     * @param {object} mygroupRes this students group
+     * @param {object} rankedgroupsRes this students list of candidates
+     */
+    function updateCandidateList(mygroupRes, rankedgroupsRes) {
+        const sortedGroup = rankedgroupsRes.sort((a, b) => b.value - a.value);
+        const table = document.getElementById("candidatesTable");
+        updateCandidateTable(table, sortedGroup, mygroupRes);
+    }
+
+    /**
+     * @summary Updates candidate table to new candidate list and group
+     * @param {HTMLTableElement} table table HTML element to update
+     * @param {object[]} candidateList Sorted array of candidates
+     * @param {object} thisGroup The groups whose candidates this is
+     */
+    function updateCandidateTable(table, candidateList, thisGroup) {
+        window.utiljs.clearChildren(table);
```

Comment:
Import denne funktion i stedet for you know  `const {clearChildren} = window.utiljs`

Reply:
imported 👍